app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -631,7 +634,7 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
 
     // We should handle the gesture whenever the system does not handle it.
     bool isHandActionEnabled = systemGestureDetected && (!systemTakesOverWhenHandsFacingHead || mHandeness == Left);
-    delegate.SetHandJointLocations(mIndex, jointTransforms);
+    delegate.SetHandJointLocations(mIndex, jointTransforms, jointRadii);
```

Comment:
... and finally use `std::move(jointRadii)` here.

Double and triple check that `jointRadii` is not used in this method after the `move`